### DISCUSSION TITLE
wire:loading not hidden at page load

### DISCUSSION THREAD

**Discussion by michabbb** [livewire/livewire#4726] [1. Help] [answered]
hi there,

i don´t know if that is intended to be that way, so I am asking because when looking at the [docs](https://laravel-livewire.com/docs/2.x/loading-states) I don´t see any "hidden" class.

in my case I have a file upload input:

`<input type="file" wire:model="upload">`

and an SVG as a spinner:

`<SVG wire:loading wire:target="upload"......>`

but I have to add the "hidden" class so it doesn´t get shown at page start.
the docs don´t explain or show this, but that´s the only way to make it work.

so i would like to ask: is the "hidden" needed in general ? or is this example something special because it´s an SVG ?

thanks!

**Comment by joshhanley** [comment, marked as answer]
@michabbb if it's not hiding on page load, maybe make sure you have Livewire's styles in your head tag and that they are actually loading on the page, as it should set `display:hidden` on any element that has `wire:loading` on it.

Hope this helps!

**Reply by michabbb** [reply]
@joshhanley thanks again for your help! I had different "app.blade.php" files and in of them `@livewireStyles` was indeed missing 🤦‍♂️ 
problem solved 😊

**Reply by joshhanley** [reply]
No worries! 🙂
